DEPS.xwalk

```diff
@@ -115,6 +115,15 @@ hooks = [
     ],
     'pattern': '.',
     'name': 'upstream_revision'
+  },
+  {
+    'action': [
```

Comment:
Can you add a "TODO(darktears): Remove when we move to M45 since the patch is part of Skia there"?